elastalert/config.py

```diff
@@ -372,7 +372,7 @@ def create_alert(alert, alert_config):
             alert_field = [alert_field]
 
         alert_field = [normalize_config(x) for x in alert_field]
-        alert_field = sorted(alert_field, key=lambda (a, b): alerts_order.get(a, -1))
+        alert_field = sorted(alert_field, key=lambda (a, b): alerts_order.get(a, 1))
```

Comment:
Maybe consider a future enhancement to make alert ordering semantics configurable within a rule?